setup.cfg

```diff
@@ -1,5 +1,18 @@
+[metadata]
+name = Tubes
+version = 0.2.0
+description = Flow control and backpressure for event-driven applications.
+license = MIT
+url = https://github.com/twisted/tubes/
+
+[options]
```

Comment:
While you're on it, maybe add `python_requires` too? Could be in a follow-up PR tho.

Reply:
I think tubes still supports python 2 (hence the six dep)